lib/sparql_to_sw_solr/instance_solr_doc.rb

```diff
@@ -58,6 +61,7 @@ def instance_uri_to_ckey
     def values_from_solutions(solutions, predicate_name)
       values = []
       solutions.each_solution do |soln|
+        next unless soln.bindings.keys.include?(:o) && soln.bindings.keys.include?(:p)
```

Comment:
Why this skip?  Does the solution object always use the same `:o` and `:p` keys or can a query change that behavior?  If the query changes it, does it matter?